src/viewElements/shared/listView/ListView.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Header, Item, Icon, Input, Button, Text, Container, Content, ListItem, View } from 'native-base';
+import { FlatList } from 'react-native';
+import I18n from 'yasav/locales/i18n';
+import StyleList from 'yasav/src/styles/List'
+import { ActivityTypeEnum } from 'yasav/src/const';
+
+export default class ListView extends React.Component {  
+
+  constructor(props) {
+    super(props);
+    this.renderRow = this.renderRow.bind(this);
+  }
+
+  i = 0
```

Comment:
Renaming the "i" (/"this.i") var to something more explicit ?
Like "listItemNumber" / "listItemCount" / "listItemParity"